sklearn/datasets/tests/test_common.py

```diff
@@ -94,7 +94,8 @@ def _generate_func_supporting_param(param, dataset_type=("load", "fetch")):
                 condition=name.startswith("fetch") and _has_network(),
                 reason="Skip because fetcher requires internet network",
             )]
-            marks.append(markers_fetch.get(name, pytest.mark.basic))
+            if name in markers_fetch:
+                marks.append(markers_fetch[name])
```

Comment:
@glemaitre maybe I'm misunderstanding [the link you provided](https://docs.pytest.org/en/stable/example/parametrize.html#set-marks-or-test-id-for-individual-parametrized-test), but shouldn't the `pytest.mark.network` be added here? The network marker is defined in [`conftest.py`](https://github.com/scikit-learn/scikit-learn/blob/9cd13a1fa16708f94a0d821ac2865fa7d981cad8/conftest.py#L90).

Reply:
The network is the line just above. But we are going to refactor this part with https://github.com/scikit-learn/scikit-learn/pull/17553